JesusFilm/core
# fix: add rtl to stories

# Description

Add RTL logic to the theme provider. By doing so we're fixing the missing configuration for the set up of MUI RTL. And we're also setting up the Stories to be supporting RTL.

A continuation/fix for this todo. It's missing the RTL config on the ThemeProvider component
[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29688713/todos/5413044377)

# How should this PR be QA Tested?

- [ ] Default theme to include an RTL story

After running the storybook server. And in any Journey Blocks stories within Journeys UI
1. Create a new story with a proper RTL description
2. Set rtl to be true on the new story parameters e.g. `RightToLeftButton.parameters = { rtl: true}`
3. Check story on storybook

- [ ] Created story to be in RTL

# Checklist
